index.html

```diff
@@ -83,6 +83,7 @@ <h3 id="fondamenti-di-elettronica">Fondamenti di elettronica</h3>
 <ul>
   <li>Appunti di lezioni ed esercitazioni di Francesco Bossio <a href="https://github.com/valerionew/triennale-elettronica-polimi/tree/master/2/FDE/appuntiBossio"> Cartella</a></li>
   <li>Appunti di Giacomo Tombolan del corso - Download <a href="2/FDE/appuntiTombolan/1_TOMBOLAN_FDE_2018_APPUNTI.pdf" download>Parte 1</a> - <a href="2/FDE/appuntiTombolan/22_TOMBOLAN_FDE_2018_APPUNTI.pdf" download> Parte 2</a> - <a href="2/FDE/appuntiTombolan/38_TOMBOLAN_FDE_2018_APPUNTI.pdf" download> Parte 3</a> - <a href="2/FDE/appuntiTombolan/53_TOMBOLAN_FDE_2018_APPUNTI.pdf" download> Parte 4</a> - <a href="2/FDE/appuntiTombolan/73_TOMBOLAN_FDE_2018_APPUNTI.pdf" download> Parte 5</a></li>
+    <li><a href="2/FDE/DomandeOrale/domande_elettronica.md" download>Domande fatte all'orale</a></li>  
```

Comment:
il link al file sarebbe da fare come quello delle domande di Elettromagnetismo, in modo che si apra la versione web rendereizzata da github, così come è ora il file markdown viene scaricato sul pc dell'utente e poi l'utente deve renderizzarlo. Per reference:
<a href="https://github.com/valerionew/triennale-elettronica-polimi/blob/master/2/EEO/domandeTeoria.md">Link</a>